approaches/bulk-data-loading/src/VendiaClient.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+export class VendiaClient {
```

Comment:
Could/should we use our client SDK for this?

Reply:
Yes, great call. I incorporated Client SDK everywhere and removed VendiaClient.js entirely.